prerequisites/build-functions/edit_GCC_download_prereqs_file_if_necessary.sh

```diff
@@ -14,7 +14,7 @@ replace_wget()
     sed -i${backup_extension} -e '2 a\'$'\n'". ${OPENCOARRAYS_SRC_DIR}/prerequisites/build-functions/ftp_url.sh"$'\n' "${download_prereqs_file}"
```

Comment:
I'm reasonably confident it's correct, but it's necessary only for GCC versions < 7 and is therefore performed only for such versions.   My first-ever GCC patch was approved and applied to the trunk so that the GCC 7 version of the [download_prerequisites] script uses `curl` when `wget` is not available.  The only time ftp would need to be substituted would be if neither `curl` or `wget` are available, which is probably rare so I wouldn't worry about this.

[download_prerequisites]: https://github.com/gcc-mirror/gcc/blob/master/contrib/download_prerequisites#L58